share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/memory.param

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2013-2021 Axel Huebl, Rene Widera, Benjamin Worpitz
```

Comment:
@sbastrakov Did you apply any changes to this file or is this the default file? If yes you could add yourself as an author - if not, please remove the file.

Reply:
Yes, I had to add this file as I want supercell size to be multiple of 4 but not necessarily 8. Can add myself, but I thought it's not a substantial change

Reply:
You should, so the others now whom to ask if there are any questions regarding file changes for this specific setup.